src/common/map/MapService.js

```diff
@@ -120,6 +120,53 @@
     };
   })();
 
+  function bboxTransfom(coordinates, oldProjection, projection) {
+    if (!coordinates) {
+      return [[]];
+    }
+    return [[
+      ol.proj.transform([coordinates[0], coordinates[1]], oldProjection, projection),
+      ol.proj.transform([coordinates[0], coordinates[3]], oldProjection, projection),
+      ol.proj.transform([coordinates[2], coordinates[3]], oldProjection, projection),
+      ol.proj.transform([coordinates[2], coordinates[1]], oldProjection, projection)
+    ]];
+  }
+
+  function createBboxVector(coordinates, projection) {
+    var geojsonObject = {
+      'type': 'FeatureCollection',
```

Comment:
We don't need a `FeatureCollection` here. A simple `Feature` does the job as well.